### DISCUSSION TITLE
How to create multi tenant App using goFr

### DISCUSSION THREAD

**Discussion by primadi** [gofr-dev/gofr#690] [Q&A] [answered]
Hi all,

I want to create multi tenant App, with strategy :  one db connection is for one tenant.
It is ok, if i have to manage myself, correlation of tenant and db Connection.

But, can i have db connection dynamically ?

Tenant ID will save on X-Tenant header, and then, based on tenant ID, i will get db connection from db pool.
How to handle this requirement using GoFr ?

Thank you.

**Comment by Umang01-hash** [comment, marked as answer]
Currently, GoFr doesn't support multi-tenant db functionality but our team will review the requirements and see if it aligns with our development path.
